src/gvmd.c

```diff
@@ -171,14 +171,14 @@
 /**
  * @brief Scanner port.
  *
- * Used if /etc/services "otp" and --port missing.
+ * Used if --scanner-port is missing.
  */
 #define OPENVASSD_PORT 9391
 
 /**
  * @brief Manager port.
  *
- * Used if /etc/services "gmp" and --sport are missing.
+ * Used if /etc/services "otp" and --port are missing.
```

Comment:
Yes "otp" is correct here.  If you look further down in manager_listen there
is a call to getservbyname with arg "otp".  This looks for the "otp" entry in
/etc/services (the services database).

In the very distant past the scanner was running on 9390 and we registered
9390 as "otp".  When Manager became the interface for client applications
we switched Manager to 9390, but we didn't update the /etc/services entry
(because it's a pain, or maybe we lost contact with the person who
registered it).

So we just forgot to update these comments when we switched Manager
to use the "otp"port.

Reply:
I wasn't sure if this was something that could be changed simply because my services list does not contain "otp", so it looks like it has to be manually configured anyway.

However, after looking it up, "otp" is still the official service name in the IANA port number registry, so it makes sense to keep using it.